app/components/UI/Notification/TransactionNotification/index.js

```diff
@@ -209,7 +209,8 @@ function TransactionNotification(props) {
 			const existingGasPrice = new BigNumber(tx?.transaction?.gasPrice || '0x0');
 			const gasFee = existingGasPrice
 				.times(transactionAction === ACTION_CANCEL ? CANCEL_RATE : SPEED_UP_RATE)
-				.toString();
+				.toString()
+				.split('.')[0]; // strips decimals if any, coming from the 'times' operation
 			setGasFee(gasFee);
```

Comment:
instead of using `.split` we could use `.substring`:

```suggestion
			const gasFeeValue = gasFee.toSrting();
			const [ from, to ] = [ gasFeeValue.indexOf('.'), 0 ];
			const gasFeeSubString = gasFeeValue.substring(from, to);
			setGasFee(gasFeeSubString);
```

the result is the same without having to split the string (an operation that can be costly depending on the situation). I also just find this a bit easier to grok, but maybe that's just me? this could also be a function that's added and tested in the `number/index.js` util (similar to what you do with the creation of `safeNumberToBN`).